### ISSUE TITLE
Feature request: jump method

### ISSUE THREAD

**Issue by YTaoger** [hrynko/vue-pdf-embed#138]
# For example, as the browser browsed pdf components by default, you could go to a specific page and scroll to other pages, rather than having to change the page and not scroll to other pages of pdf.
<img width="1197" alt="image" src="https://user-images.githubusercontent.com/35984856/235043472-3378faea-6bd8-4d62-83da-55c499370db3.png">

**Comment by YTaoger** [comment]
Also, as I scroll, how do I know which page belongs to the total number of pages

**Comment by hrynko** [comment]
Hi @YTaoger,

Sorry, I don't really understand your message. Could you rephrase,

**Comment by YTaoger** [comment]
> Hi @YTaoger,
> 
> Sorry, I don't really understand your message. Could you rephrase?

I'm sorry, maybe it's my expression.
Currently, if page is not specified, all pages are viewed. But can you use parameters or methods to jump to a specific page while browsing? And, when scrolling, you can get which page is currently visible in the window?

**Comment by YTaoger** [comment]
https://user-images.githubusercontent.com/35984856/235081877-5486260e-57df-4207-a2c5-506ed54c24ef.mp4

@hrynko Excuse me, in the video, notice the change of the top page, according to the page scroll, the current page number will also change, how to achieve this effect in `vue-pdf-embed`? You can also switch the current page number.
